src/components/DefinitionList/DefinitionList.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+
+import {ClipboardButton, Link} from '@gravity-ui/uikit';
+
+import {HelpPopover} from '../HelpPopover';
+import type {HelpPopoverProps} from '../HelpPopover';
+import {block} from '../utils/cn';
+
+import {isUnbreakableOver} from './utils';
+
+import './DefinitionList.scss';
+
+type DefinitionListItemNote = string | HelpPopoverProps;
+
+export interface DefinitionListItem {
+    name: string;
+    key?: string;
+    href?: string;
+    content?: React.ReactNode;
+    title?: string;
+    copyText?: string;
+    note?: DefinitionListItemNote;
+    multilineName?: boolean;
+}
+
+export interface DefinitionListProps {
+    items: DefinitionListItem[];
+    copyPosition?: 'inside' | 'outside';
+    responsive?: boolean;
+    nameMaxWidth?: number;
+    contentMaxWidth?: number | 'auto';
+    className?: string;
+    itemClassName?: string;
+}
+
+export const b = block('definition-list');
+
+function getTitle(title?: string, content?: React.ReactNode) {
+    if (title) {
+        return title;
+    }
+
+    if (typeof content === 'string' || typeof content === 'number') {
+        return String(content);
+    }
+
+    return undefined;
+}
+
+function getNoteElement(note?: DefinitionListItemNote) {
+    let noteElement = null;
+    const popoverClassName = b('item-note-tooltip');
+    if (note) {
+        if (typeof note === 'string') {
+            noteElement = (
+                <HelpPopover
+                    className={popoverClassName}
+                    htmlContent={note}
+                    placement={['bottom', 'top']}
+                />
+            );
+        }
+
+        if (typeof note === 'object') {
+            noteElement = <HelpPopover className={popoverClassName} {...note} />;
+        }
+    }
+    return noteElement;
+}
+
+export function DefinitionList({
+    items,
+    responsive,
+    nameMaxWidth,
+    contentMaxWidth = 'auto',
+    className,
+    itemClassName,
+    copyPosition = 'outside',
+}: DefinitionListProps) {
+    const keyStyle = nameMaxWidth
+        ? {
+              flexBasis: nameMaxWidth,
+          }
+        : {};
+
+    const valueStyle =
+        typeof contentMaxWidth === 'number'
+            ? {
+                  flexBasis: contentMaxWidth,
+                  maxWidth: contentMaxWidth,
+              }
+            : {};
+    return (
+        <dl className={b({responsive}, className)} role="list">
+            {items.map(({name, href, content, title, copyText, note, key, multilineName}) => {
+                const term = href ? <Link href={href}>{name}</Link> : name;
```

Comment:
Maybe instead of having `href` prop we allow `name` to be ReactNode to render whatever user want?

Reply:
Like it! But in this case `key` property should be required.

Reply:
We can generate them by ourselves